server/config.py

```diff
@@ -26,23 +28,40 @@
 KEY_SECURE_SESSION_COOKIES = 'secure_session_cookies'
 KEY_SAMESITE_SESSION_COOKIES = 'samesite_session_cookies'
 
+from distutils.util import strtobool
 
 #
 # Container for properties that should be read on startup, and maybe be
 # refreshed dynamically.
 #
-class Config:
+# If using redis all the keys are cached in the redis server, otherwise
```

Comment:
Can you add comments that explain how/when you are getting the values from the datastore entity?